SDVModTest/UIElements/ExperienceBar.cs

```diff
@@ -72,25 +97,46 @@ public void Dispose()
             _timeToDisappear.Dispose();
         }
 
+        public void optionChanged(string identifier, bool isOn)
+        {
+            switch (identifier)
+            {
+            case OptionKeys.ShowLevelUpAnimation:
+                ToggleLevelUpAnimation(isOn);

```

Comment:
Why not just use these as the callbacks? I honestly don't see the point in having the identifier.

Reply:
good point. although the function parameters don't match, and I havent found a way to do so.